docs/_build/html/_sources/methods.rst.txt

```diff
@@ -0,0 +1,444 @@
+.. set up matlab code highlighting
+.. highlight:: matlab
+
+.. set up referencing
+.. _methods:
+
+Methods
+=======
+
+This documentation will focus on using ``xolotl`` in ``MATLAB``, though it is
+possible to compile and run the ``C++`` code on its own.
+
+A ``xolotl`` object can be instantiated ::
+
+  x = xolotl;
+
+For consistency, this documentation will assume a ``xolotl`` object named ``x``.
```

Comment:
it's not "for consistency". just say "in the rest of this documentation, we will assume,...."

Reply:
done